lib/neworder.rb

```diff
@@ -0,0 +1,30 @@
+require_relative './menu.rb'
+require_relative './timestamp.rb'
+require_relative './sms.rb'
+
+class NewOrder
+
+  attr_reader :menu, :order, :sms
+
+  def initialize
+    @order = []
+    @menu = Menu.new
+    @sms = SMS.new
+  end
+
+  def add_dish(dish, number)
+    number.times { @order << { dish => @menu.items[dish] } }
+  end
+
+  def total
+    @amount = 0.00
+    @order.each { |item| @amount += item.values[0] }
+    @amount
```

Comment:
You could also write this as:

```ruby
def total
  @order.map { |item| item.values[0] }.sum
end
```

Also — no need for `@amount` to be an instance variable right? Since it's not used elsewhere?